src/hooks/use-confirm.tsx

```diff
@@ -34,11 +42,19 @@ const useConfirm = (): UseConfirmResult => {
   const checkConfirmation = ({
     message,
     title,
+    confirmBtn,
```

Comment:
use the whole words, not "btn", "msg" etc, and you use it as content for the button do so we can put in it not only string for translation but anything we want

Reply:
fixed